website/content/docs/enterprise/license/faq.mdx

```diff
@@ -0,0 +1,124 @@
+
+Consul server agents will detect the absence of the license and immediately exit.
+
+Consul client agents will attempt to retrieve the license from servers if certain conditions are met: ACLs are enabled, a ACL token is provided to the client agent, the client agents configuration contains `start_join/retry_join` addresses, the start/retry join addresses are addresses of the Consul servers.
+
+Consul snapshot agents will attempt to retrieve the license from servers if certain conditions are met: ACLs are enabled, a ACL token is provided to the client agent, the client agents configuration contains `start_join/retry_join` addresses, the start/retry join addresses are addresses of the Consul servers.
+
+## Q: Where can users get a trial license for Consul Enterprise?
+
+Visit https://www.consul.io/trial for a free 30-day trial license.
+
+## Q: Are the license files locked to a specific cluster?
+
+The license files are not locked to a specific cluster or cluster node. The above changes apply to all nodes in a cluster.
+
+## Q: Will this impact HCP Consul?
+
+This will not impact HCP Consul.
+
+## Q: Does this need to happen every time a node restarts, or is this a one-time check?
+
+Consul Enterprise binaries starting with v1.10.0+ent, will be subject to EULA check. Release v1.10.0+ent introduces the EULA check for eval licenses (non-EVAL licenses already go through EULA check during contractual agreement).
+
+The agreement to a EULA happens only once (when the user gets their license), Consul Enterprise **will check for the presence of a valid license every time a node restarts**.
+
+When a customer upgrades existing clusters to a v1.10.0+ent release, they need to have a valid license to successfully upgrade. This valid license must be auto-loaded.
+
+When a customer deploys new clusters to a v1.10.0+ent release, they need to have a valid license to successfully upgrade. This valid license must be on-disk (auto-loaded).
+
+## Q: What are the scenarios that a customer must plan for because of these changes?
+
+New Consul cluster deployments using v1.10.0+ent will need to have a valid license to successfully deploy.
+This valid license must be on-disk (auto-loaded) or as an environment variable.
+
+## Q: What is the migration path for customers who want to migrate from their existing license-as-applied-via-the-CLI flow to the license on disk flow?
+
+Run consul `license get -signed` to extract the license from their running cluster. Store the license in a secure location on disk.
+Set up the necessary configuration so that when Consul Enterprise reboots it will have access to the required license. This could be via the client agent configuration file or an environment variable.
+Visit the [Enterprise License Tutorial](https://learn.hashicorp.com/tutorials/nomad/hashicorp-enterprise-license?in=consul/enterprise) for detailed steps on how to install the license key.
+Proceed with the upgrade as normal.
```

Comment:
```suggestion
1. Set up the necessary configuration so that when Consul Enterprise reboots it will have access to the required license. This could be via the client agent configuration file or an environment variable.
1. Visit the [Enterprise License Tutorial](https://learn.hashicorp.com/tutorials/nomad/hashicorp-enterprise-license?in=consul/enterprise) for detailed steps on how to install the license key.
1. Proceed with the upgrade as normal.
```